gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/sampling/functions/VertexDegree.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.sampling.functions;
+
+import java.security.InvalidParameterException;
+
+/**
+ * Keeps the types of vertex-degrees and the respective property names.
+ */
+public class VertexDegree {
```

Comment:
IMO the whole class should be an enum. Providing fromString, getName methods. Furthermore it would reduce the boilerplate VertexDegree.DegreeType to VertexDegree.InputDegree. 
Even though I would prefere something easier like IN, OUT, INOUT since degree is also part of the Enum or class name.